scripts/install_environment.sh

```diff
@@ -146,20 +147,40 @@ EOF
     chmod +x "${hook_filename}"
 }
 
+function install_podman(){
+    sudo systemctl disable --now podman.socket || true
+    sudo rm -rf /run/user/${UID}/podman
+    sudo rm -rf /run/podman
+    sudo dnf install podman -y
+    sudo systemctl enable --now podman.socket
+    systemctl --user enable --now podman.socket
+    sudo loginctl enable-linger $USER
+}
+
 function install_runtime_container() {
     echo "Installing container runtime package"
-    if ! [ -x "$(command -v docker)" ] && ! [ -x "$(command -v podman)" ]; then
-        sudo dnf install podman -y
-    elif [ -x "$(command -v podman)" ]; then
-        current_version="$(head -n1 <(podman version) | awk '{print $2}')"
-        minimum_version="1.6.4"
-        if ! version_is_greater "$current_version" "$minimum_version"; then
-            sudo dnf install podman-$minimum_version -y
-        fi
+    if [ -x "$(command -v docker)" ]; then
+      echo "docker is already installed"
+      return
+    fi
+
+    if [ ! -x "$(command -v podman)" ]; then
+      install_podman
```

Comment:
if on some environment (and I think it's most local environments) we have podman installed but without the podman socket, then the podman socket will not be configured
maybe we should check if there's an active socket and based on that do or don't do the socket configuration